### ISSUE TITLE
Wood and Stone Construct Hammer Arm Unplaceable

### ISSUE THREAD

**Issue by nekhs** [Mithion/Mana-And-Artifice#1024]
**Describe the bug**
Can't place Wooden or Stone Construct Hammer Arm on either the gantry or an armor stand.

**To Reproduce**
1. Get item Wooden or Stone Construct Hammer Arm (left or right does not matter)
2. Use on gantry or armor stand
3. Have tier 2
4. arm does not place and therefore cannot create construct with wooden hammer arm
5. Weirdly, Wickerwood Hammer Arm does place successfully

**Expected behavior**
Place arm on construct gantry or armor stand and add to construct.

**Screenshots**
![hammer arm fails to place](https://github.com/user-attachments/assets/df0e15fc-833d-431d-ae64-fab0ab024152)

**Version Information (please complete the following information):**
 - Mod Version: 3.1.0.1-all
 - Forge Version 47.3.12

**Additional context**
I'll admit I'm playing a kitchen sink pack I threw together, but I don't think there's anything that would alter the behavior of the gantry in it. Modpack link is https://www.curseforge.com/minecraft/modpacks/yeet-the-fourth

**Comment by ShelLuser** [comment]
I'm playing with the exact same setup (though still using Forge 47.3.0) and I/we can't reproduce this:

![image](https://github.com/user-attachments/assets/564b3b35-445f-43c1-a02c-356509ecc121)

I'm in creative, but my friend plays survival.. as you can see, they can easily attach the stone hammer arm onto an armor stand without any issues. Do your logs mention anything out of the ordinary by any chance?

So basically... we placed an armor stand, got the (stone) hammer arm and just placed it without any issues.

**Comment by groting** [comment]
[Known problem with Better Combat](https://discord.com/channels/398610095585624074/857285117734027324/1267563425404424392)

**Comment by Mithion** [comment]
From FredHoof on Discord:

[...] a simpler fix is adjusting Better Combat's fallback_compatibility JSON file to imclude "construct_hammer" as part of the blacklist.

Better Combat would need to support negative numbers for speeds, or include that into the blacklist by default in order to fix this issue permanently.
